### PR TITLE
Devnet Docs 

### PR DESCRIPTION
**Pull request by thecoderpanda**
Fixes #

### Checklist
- [x] My branch is up-to-date with upstream/main branch.
- [x] Everything works and tested for major version of Python/NodeJS/Go and above.
- [x] I ran pre-commit checks against my changes.

#### Added
Getting Started with Devnet
Troubleshooting Guide
Health Check

### CODE REVIEW

File: docs/build-with-powerloom/snapshotter-node/lite-node/getting-started.md [hunk 1]
```diff
@@ -382,12 +384,12 @@ This will clone the repository into a directory named `powerloom-testnet`.
    ./init.sh
 ```
 :::tip
- Please do not use your main wallet to run the snapshotter lite node. It is advised to create a burner wallet by using any of these tools:
+ Please do not use your NFT minting wallet to run the snapshotter lite node.The safest approach The safest approach is to generate and assign a burner wallet. You can generate a burner wallet through any of the tools listed below:
```
Comment (muku314115) [inline docs/build-with-powerloom/snapshotter-node/lite-node/getting-started.md:387]: "The safest approach" is repeated

File: docs/build-with-powerloom/snapshotter-node/lite-node/getting-started.md [hunk 2]
```diff
@@ -280,12 +282,12 @@ If you're a developer and want to play around with the code, instead of running
 </details>
 
 :::tip
- Please do not use your main wallet to run the snapshotter lite node. It is advised to create a burner wallet by using any of these tools:
+ Please do not use your NFT minting wallet to run the snapshotter lite node.The safest approach The safest approach is to generate and assign a burner wallet. You can generate a burner wallet through any of the tools listed below:
```
Comment (muku314115) [inline docs/build-with-powerloom/snapshotter-node/lite-node/getting-started.md:285]: "The safest approach" is repeated

File: docs/build-with-powerloom/devnet/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+   - `SIGNER_ACCOUNT_PRIVATE_KEY`: The private key corresponding to the burner wallet address.
+   - `SNAPSHOT_CONFIG_REPO`: https://github.com/your-username/snapshotter-configs [Change the username to your GitHub profile to point to your forked snapshotter-configs repository.]
+   - `SNAPSHOT_CONFIG_REPO_BRANCH`: Change to your own branch or use the `devnet` branch if you are just getting started.
+   - `SNAPSHOTTER_COMPUTE_REPO`:  https://github.com/your-username/snapshotter-computes [Change the username to your GitHub profile to point to your forked snapshotter-computes repository.]
+   - `SNAPSHOTTER_COMPUTE_REPO_BRANCH`: Change to your own branch or use the `devnet` branch if you are just getting started.
+   - `PROST_RPC_URL`: `https://rpc-prost1d.powerloom.io` The URL for the Powerloom Protocol Chain RPC service.
+   - `PROTOCOL_STATE_CONTRACT`: The contract address for the protocol state.
+
+:::note
+We offer two types of Protocol State Contracts, each tailored to different Epoch Sizes, to cater to the unique complexities of your application:
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/getting-started.md:88]: A lot of lines seem very GPT like tbh, better to keep it to the point. For example -
```
Currently, there are two variations of Protocol State Contracts available. One where [Epoch]<link to epoch explanation from docs> size is set to 1 and other where epoch size is set to 10 blocks.  This should allow developers to build and experiment with a variety of use cases. If you have any custom needs, reach out to the team <link>
```
Just a quick draft but you should get the idea.

File: docs/build-with-powerloom/devnet/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+   - Epoch Size 1 Contract: `0x750129050859a00bCe706B9Bb43f7117EE0c344b`
+   - Epoch Size 10 Contract: `0x5B1f374C7680B310CD68351B8e9e6BeD34d8884f`
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/getting-started.md:90]: These addresses are bound to change over time, let's not keep these here.

File: docs/build-with-powerloom/devnet/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+Consider the complexity and requirements of your application when selecting the appropriate Protocol State Contract. This will ensure optimal performance and efficiency for your specific use case.
+:::
+  
+   - `RELAYER_HOST`: `https://relayer-devnet-1d.powerloom.dev` The host address for the relayer.
+   - `NAMESPACE`: The unique key used to identify your project namespace.
+   - `POWERLOOM_REPORTING_URL`: `https://devnet-reporting.powerloom.dev` The URL for reporting to PowerLoom.
+   - `PROST_CHAIN_ID`: For Devnet, it's `104`.
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/getting-started.md:98]: Let's avoid using any values, if necessary, point them too `env.example` file

File: docs/build-with-powerloom/devnet/health-tracking.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 2
+---
+
+# Health Tracking
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/health-tracking.md:5]: Aren't these instructions exactly the same as Snapshotter Node health tracking? If yes, we should either link to those or reorganize the content hierarchy to better suit our needs. 
Don't think it is a good idea to repeat content.

File: docs/build-with-powerloom/devnet/troubleshooting.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 3
+---
+
+# Troubleshooting
+
+
```
Comment (xadahiya) [inline docs/build-with-powerloom/devnet/troubleshooting.md:7]: All these points are present in snapshotter node troubleshooting I think. We should just link to those and add devnet specific things here like issue with minting, burner wallet, how to check your submissions, etc.

File: docs/participate/devnet.md [hunk 8]
```diff
@@ -1,10 +1,51 @@
...
+
+5. **Analytical Tools**: Develop powerful analytical tools that enable in-depth analysis of blockchain data. Uncover trends, patterns, and anomalies to drive data-informed strategies and optimizations.
+
+## Get Started with Devnet
+
+Ready to embark on your Devnet journey? Follow these simple steps:
+
+1. Visit our [Mint page](https://mint-devnet.powerloom.dev) to mint your Devnet slot.
+2. Once minted, head over to the [Getting Started guide](../build-with-powerloom/devnet/getting-started.md) for a step-by-step walkthrough on setting up your development environment.
+3. Explore the Powerloom documentation to familiarize yourself with our APIs, data models, and best practices.
```
Comment (xadahiya) [inline docs/participate/devnet.md:48]: Not sure if we should write this. Our focus is not on APIs or data models, this should talk about snapshotter nodes, consensus, etc.

File: docs/participate/devnet.md [hunk 9]
```diff
@@ -1,10 +1,51 @@
...
+
+- **Comprehensive Datasets**: Powerloom provides access to rich and diverse data markets. This allows you to work with real-world data and test your applications under realistic conditions. 
+
+- **Collaborative Environment**: Devnet fosters a vibrant developer community where you can connect with other builders, share knowledge, and collaborate on projects. Leverage the collective expertise to accelerate your development efforts.
+
+## What Can You Build with Devnet?
+
+The possibilities are truly endless when it comes to building data applications on Powerloom Devnet. Here are a few exciting examples:
+
+1. **Data Dashboards**: Create interactive visualizations and dashboards that provide insights into blockchain metrics, token economies, user behavior, and more. Empower users with data-driven decision making.
```
Comment (xadahiya) [inline docs/participate/devnet.md:32]: Let's only write things that users can build today. We can expand later. 
Better to keep it simple and to the point IMO.

File: docs/participate/devnet.md [hunk 10]
```diff
@@ -1,10 +1,51 @@
...
+
+Devnet enables you to efficiently customize and analyze blockchain data to power your applications. Whether you're building a simple prototype or a complex applications, Devnet makes the development process smooth and straightforward.
+
+## Why Use Powerloom Devnet?
+
+There are several key benefits to using Powerloom Devnet for your development needs:
+
+- **Easy Onboarding**: Minting a Devnet slot is quick and simple through our user-friendly minting page. You can get started building right away without any lengthy setup process.
+
+- **Comprehensive Datasets**: Powerloom provides access to rich and diverse data markets. This allows you to work with real-world data and test your applications under realistic conditions.
```
Comment (xadahiya) [inline docs/participate/devnet.md:24]: ```Powerloom provides access to rich and diverse data markets```
Not on devnet

File: docs/participate/devnet.md [hunk 11]
```diff
@@ -1,10 +1,51 @@
...
-You can signup [here](https://airtable.com/appr3wKRsn4VoOvpi/pagxxT6QcL0OwGH0e/form), and we'll reach out to you once it goes live! 
+You can mint our Devnet slot from our [Mint page](https://devnet-mint.powerloom.dev/). Once minted, you can navigate to the [getting started](../build-with-powerloom/devnet/getting-started.md) section to kickstart your devnet journey.  
 
 :::
+
+## What is Powerloom Devnet? 
+
+Powerloom Devnet is a special development environment that allows developers to easily build and test data-driven applications on Powerloom. It provides a sandbox where you can experiment, learn, and create without impacting the main network.
+
+Devnet enables you to efficiently customize and analyze blockchain data to power your applications. Whether you're building a simple prototype or a complex applications, Devnet makes the development process smooth and straightforward.
```
Comment (xadahiya) [inline docs/participate/devnet.md:16]: We should rewrite this IMO. 
```
Whether you're building a simple prototype or a complex applications, Devnet makes the development process smooth and straightforward.
```
How does it make the process smooth and straightforward?

**Review by xadahiya — CHANGES_REQUESTED** [review]
Mostly looks good, added some comments for improvement!

**Pull request closed without merging**